docs/tutorial/workers.md

```diff
@@ -148,30 +149,83 @@ Select the flow you want to deploy, and the deployment wizard will walk you thro
 6. **Image name (my-first-deployment):** Hit `Enter` to use the default image name.
 7. **Image tag (latest):** Hit `Enter` to use the default image tag `latest`.
 8. **Would you like to push this image to a remote registry? (y/n):** Select `n` for now; we can keep this image local.
+9. **Would you like to save configuration for this deployment for faster deployments in the future? (y/n):** Select `y` to initiate a `prefect.yaml` file.
```

Comment:
Really think saving the deployment config helps people feel more grounded as they build their first deployment.